### PR TITLE
Update for capital-framework 5.1.0

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-icons to 4.4.0, adding the updating alias to the update icon

## Testing

- Run `npm install && npm update`
- Run `grunt build`
- Run `npm start`
- Open `http://localhost:4000/brand-guidelines/minicons.html` and view the update icons

### CODE REVIEW

File: _includes/identity/icons/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 | {% include icons/error.svg %} | {% include icons/error-round.svg %} | error | delete, close, remove, multiply, multiplication, x |
 | {% include icons/warning.svg %} | {% include icons/warning-round.svg %} | warning | alert, exclamation-mark |
 | {% include icons/help.svg %} | {% include icons/help-round.svg %} | help | question, question-mark |
-| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update |  |
+| {% include icons/update.svg %} | {% include icons/update-round.svg %} | update | updating(animated) |
```
Comment (marteki) [inline _includes/identity/icons/index.md]: It's confusing to me to include "animated" here when it's not technically a part of the alias. 

Options to improve:
- If we're calling attention to a specific alias for triggering an animated state, let's be explicit about that. Something like 'updating (_used for animated state_)' or something.
- If 'animated' is just actually one of the aliases, let's format it the same as the other icons by separating the list of aliases with commas.
- If we decide to keep the phrasing and formatting almost exactly as is, it's missing a space before the first parenthesis. 😎

Comment (jimmynotjim) [reply _includes/identity/icons/index.md]: Thank you, fixed and pushed.

**Review by marteki — APPROVED** [review]
Shame that the "used for animated state" can't be italicized or something. But otherwise, looks great. :+1:

**Comment by jimmynotjim** [comment]
@marteki good call, updated with that change.

**Pull request merged**